Ginger/Ginger/BusinessFlowPages_New/NewAutomatePage.xaml.cs

```diff
@@ -246,15 +247,23 @@ private void LoadBusinessFlowToAutomate(BusinessFlow businessFlowToLoad)
                     mBusinessFlow.TargetApplications.CollectionChanged += mBusinessFlowTargetApplications_CollectionChanged;
 
                     UpdateRunnerAgentsUsedBusinessFlow();
-                    xAddActionMenuFrame.Content = new MainAddActionsNavigationPage(mContext);
+                    if (MainNavigationPage == null)
+                    {
+                        MainNavigationPage = new MainAddActionsNavigationPage(mContext, mBusinessFlow.CurrentActivity.TargetApplication); 
```

Comment:
need to pass only Context in this case and child pages should pull from it whatever they need

Reply:
Removed